### ISSUE TITLE
About running on android

### ISSUE THREAD

**Issue by Sabst** [spencercarli/meteor-todos-react-native-2#23]
I agree the app builds ok on android but I could not run it though (same error using emulator or real device):

```
Can't find variable: LocalCollection
selector.js:23
```

This must be about minimongo-cache/src/selector.js:
`LocalCollection = {};`

Do you see any reason why this could fail?
I could not find any related information in the related projects (ddp-client, minimongo-cache, node-ddp-client...) and I did not see much in the logs.

I noticed minimongo-cache used by node-ddp-client is quite outdated (0.0.25)... I manually upgraded it to 0.0.41 (the first version pulling babel-preset-es2015) or 0.0.48 (the latest version) but I had a new error: `cannot send a message. Unknown WebSocket id 0 in WebSocketModule.java:173`

Any idea?

**Comment by spencercarli** [comment]
This error just bit me on a client app - not sure what the fix is at this time. What version of RN are you running? 0.20, 0.21, 0.22?

I'll try and make an update in a timely fashion... as soon as I figure out what the issue is.

**Comment by Sabst** [comment]
I am using 0.21 and same error with 0.22.

**Comment by Sabst** [comment]
In case that helps, I had more success with this project: https://github.com/JustMeteor/friends
(but it is using an older RN version)

**Comment by spencercarli** [comment]
Could you try the latest version of master?

**Comment by Sabst** [comment]
Thanks, this is working now!
Even using an emulator/avd I had to change localhost to my server IP in ddpClient.js (according to the adb logs, this is the reason for the WebSocket error).

**Comment by spencercarli** [comment]
Awesome! And yeah Android doesn't work with `localhost`... which is a pain when putting together these cross platform examples :)

**Comment by Sabst** [comment]
FYI, this works (as documented in RN docs) with android physical devices too without any problem, both in debug and release modes.

_As I did not try the iOS version, does the touch/tap action work randomly on iOS too? On android, it is ignored most of the time. When the mouse is used (android emulator), this works perfectly. I remember there used to be a well known issue with onTouchTap but I thought it was fixed in the version we are using now. Or maybe there were many issues related to that!
I understand this may not be a problem with the todo app itself but wondered if this was specific to android or not._
